package.json

```diff
@@ -12,17 +12,19 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "deploy": "ember deploy production",
```

Comment:
Currently, we don't use this script in CI. I'm hoping that we update the workflow to automatically deploy the GitHub Pages site in a future PR.